src/core/devconfig.ts

```diff
@@ -119,16 +128,12 @@ export async function reconfigureQuarto(
   installed: DevConfig | null,
   source: DevConfig,
 ) {
-  const configureScript = isWindows()
-    ? ".\\configure.cmd"
-    : "./configure.sh";
-
   const quartoDir = Deno.realPathSync(
     join(quartoConfig.sharePath(), "..", ".."),
   );
 
   const process = Deno.run({
-    cmd: [configureScript],
+    cmd: ["bash", "configure.sh"],
     cwd: quartoDir,
   });
```

Comment:
I wonder is this would cause an issue when reconfigure would be called outside of the context of Git bash or MSYS. 

`reconfigureQuarto()` is called in `quarto.ts` which I think could be called from `quarto.cmd` at some point. I did not manage to trigger this to test, but still trying to get a sense of if this could cause issue. 

It is just that calling `bash` in Powershell or CMD when WSL is installed will open WSL bash window - so run in Ubuntu. Not what we want I think

Reply:
I would call this the Achilles heel of this PR, and this must be solved before this work can be viable. Thanks for raising this issue.

Reply:
As we are trying to target dev mode only mostly (as only developer would run the configure.sh after a `git clone` and not use pre-release only for testing), maybe a simple workaround would be to ask for more configuration.

We could ask the Windows developer to set in its environment an env var to the Windows bash to use, and check this before trying a reconfiguration ? 

This is quite manual and would be part of prerequisite to opt-in the feature, but it would allow the script to call the right shell with the script. 

Does it make sense ? 

I am not use on Windows to run bash script really - I usually stay in BATCH or powershell.